src/binary-search-tree.js

```diff
@@ -2,6 +2,26 @@
 /* eslint-disable global-require */
 /* eslint-disable no-unused-vars */
 /* eslint-disable no-trailing-spaces */
+// const Queue = require('./queue-helper');
+
+class Queue {
```

Comment:
This Queue class should be imported from its own file, not pasted into this file.